### ISSUE TITLE
API limits

### ISSUE THREAD

**Issue by cpsubrian** [propublica/congress-api-docs#49]
Hi, I'm in the process of replacing a GovTrack-based integration with ProPublica's API. The relatively low (5000 per day) rate limit really worries me. How hard/soft is this limit? Is there any way to raise it (even if $$)?

**Comment by dwillis** [comment]
Hi @cpsubrian, can you say more about your usage? What kinds of data are you accessing and how frequently?

**Comment by cpsubrian** [comment]
Our client scores legislators according to voting history. When a roll call or bill sponsorship happens that they they wish to score, our (Drupal) backend helps them import the appropriate data (legislator details, roll call details, bill details, votes, etc). 90% of this data is then cached in Drupal nodes, so no further API requests are necessary for end-user viewing of the scores. Typically these roll calls are added fairly infrequently, once every couple weeks or so, but occasionally they happen in batches. Also, cron-jobs may make API calls to make sure data is up-to-date (primarily bill sponsors).

So, while normal usage should stay within the cap (especially if I go through and make sure end-user requests are cached on our end), but I can't guarantee it won't. What I need to avoid is the client getting errors or white-screens due to API limits that occur from normal usage.

Prior to switching to ProPublica, we maintained our own rsync of the GovTrack data and imported that into Mongo so we could slam it as much as we want. Given problems I've had setting up the `congress` scripts myself, I opted to see if the ProPublica API could solve our needs in a more maintainable way.
